### PR TITLE
Documentation landing page updates

### PR DESCRIPTION
**Pull request by chester-leung**

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -2,11 +2,27 @@
...
+
+.. image:: images/coopetition.png
+   :scale: 80%
+   :alt: Coopetition
+   :align: center
+
+With Secure XGBoost, a party or mutually distrustful federation can outsource computation on sensitive data to an untrusted cloud service without concern for information leakage. Secure XGBoost takes the encrypted sensitive data that is transferred to the untrusted cloud and loads it into a secure enclave, where it is then decrypted and computed upon. Assuming trust in the hardware vendor, enclaves provide a trusted execution environment even in the presence of a malicious host that has compromised nearly the entire software stack, including the operating system, the hypervisor, and other processes on the same machine. Since sensitive data sent over the network is encrypted and not seen in plaintext until it's inside a trusted enclave, the data of any party is not seen by any other entity. Furthermore, all computation happens inside an enclave, so all intermediate calculations that could otherwise leak some attributes of the data are also hidden.
```
Comment (chester-leung) [inline doc/index.rst]: We should find a better way to say what I'm trying to say in the last sentence here. Any ideas?

File: doc/about.rst [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+.. image:: images/coopetition.png
+   :scale: 80%
+   :alt: Coopetition
+   :align: center
+
+With Secure XGBoost, a party or mutually distrustful federation can outsource computation on sensitive data to an untrusted cloud service without concern for information leakage. Secure XGBoost takes the encrypted sensitive data that is transferred to the untrusted cloud and loads it into a secure enclave, where it is then decrypted and computed upon. Assuming trust in the hardware vendor, enclaves provide a trusted execution environment even in the presence of a malicious host that has compromised nearly the entire software stack, including the operating system, the hypervisor, and other processes on the same machine. Since sensitive data sent over the network is encrypted and not seen in plaintext until it's inside a trusted enclave, the data of any party is not seen by any other entity. Furthermore, all computation happens inside an enclave, so all intermediate calculations that could otherwise leak some attributes of the data are also hidden.
```
Comment (chester-leung) [inline doc/about.rst]: We should find a better way to say what I'm trying to say in the last sentence here. Any ideas?

Comment (podcastinator) [inline doc/about.rst]: > Assuming trust in the hardware vendor, enclaves provide a trusted execution environment...

Rephrase this sentence to: "Enclaves provide a trusted execution environment that protects the confidentiality and integrity of all code and data loaded within the enclave, even in the presence of a malicious host that has compromised the entire software stack outside the enclave (i.e., the operating system, the hypervisor, or other processes on the same machine).

Comment (podcastinator) [reply doc/about.rst]: I think we can drop the last sentence, it's redundant given we already say that enclaves provide a trusted execution environment.

File: doc/about.rst [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+.. image:: images/parties.png
+   :scale: 40%
+   :alt: Architecture 
+   :align: center
+
+Unfortunately, enclaves are vulnerable to side channel attacks. Prior work has shown that significant information can be inferred just from side channels [`1 <https://www.microsoft.com/en-us/research/wp-content/uploads/2016/02/MSR-TR-2015-70.pdf>`_, `2 <https://www.ieee-security.org/TC/SP2015/papers-archived/6949a640.pdf>`_]. As a result, Secure XGBoost provides the option of oblivious training and inference, leveraging algorithms that execute independently of input data and therefore do not leak access patterns. Oblivious algorithms protect against many side channel attacks, adding another layer of security to the system.
```
Comment (podcastinator) [inline doc/about.rst]: Amend this paragraph to:

Unfortunately, enclaves are vulnerable to side-channel attacks, and researchers have shown that attackers can infer a significant amount of information about the data within the enclave by simply observing the memory access patterns of the enclave program [`1 <https://www.microsoft.com/en-us/research/wp-content/uploads/2016/02/MSR-TR-2015-70.pdf>`_, `2 <https://www.ieee-security.org/TC/SP2015/papers-archived/6949a640.pdf>`_]. Attacker scan observe these access patterns in a variety of ways, such as snooping on the memory bus, monitoring the page table, or via cache-timing attacks.

To address this problem, Secure XGBoost provides the option of "data-oblivious" training and inference -- i.e., it leverages specially-designed algorithms whose memory access patterns are _independent_ of the input data, and hence, do not leak any information about the data within the enclave. Data-oblivious algorithms prevent all side-channel attacks induced by access-pattern leakage, and add another layer of security to the system.

**Review by podcastinator — CHANGES_REQUESTED** [review]
Thanks! Left a few comments.

File: doc/about.rst (hunk 3 above)
Comment (chester-leung) [reply doc/about.rst]: Done

**Review by podcastinator — APPROVED** [review]
Thanks!

**Pull request merged**